site/app/Resources/views/user/settings_notification.html.twig

```diff
@@ -0,0 +1,21 @@
+{#
+Template for the user to edit notification settings
+page path: /user/settings/notifications
+#}
+{% extends 'user/settings_base.html.twig' %}
+
+{# Inform settings_base.html.twig about the page currently displayed #}
+{% set settings_subpage = 'notifications' %}
+
+{% block settings_content %}
+    <h2>Notification Settings</h2>
+    <p> By default, all users get notifications from the LibreCores community in the form of web notifications
+        on the website as well as email notifications to their registered email. The settings for each of these notifications
+        is down to personal preferences and can be edited in the panel down below</p>
+    {# Set global status of user subscription to emails beforehand #}
+    {% set enStatus = user.subscribedToEmailNotifications ? "SUBSCRIBED" : "NOT SUBSCRIBED" %}
+    <h3>Email Notifications ({{ enStatus }})</h3>
```

Comment:
You don't need that enStatus part, the form will already show the current enabled/disabled state.